extensions/amp-social-share/0.1/test/test-amp-social-share.js

```diff
@@ -292,6 +315,21 @@ describes.realWin(
       });
     });
 
+    it('opens sms: window in _top on iOS>=13 Webkit', () => {
```

Comment:
Is there a way of actually testing that mail/sms in iOS works and doesn't error as opposed to asserting that for a specific version we call it with `_top`? Ideally, we'd have a test that breaks if this feature breaks in a future version of `iOS`.

Reply:
Tracking through #25284